proto/config/netconfig.proto

```diff
@@ -26,6 +26,17 @@ message NetworkConfig {
 
   // wireless specification
   WirelessConfig wireless = 10;
+
+  // Maximum transmission unit (MTU) to set on the associated network adapter(s).
+  // This is the largest IP packet that the underlying link can and is allowed to carry.
+  // This does not include the L2 header size (e.g. Ethernet header or a VLAN tag size).
+  // The value is a 16-byte unsigned integer, representing the MTU size in bytes.
+  // The minimum legal value for the MTU is 68 (RFC 791).
```

Comment:
Shouldn't we specify 1500 or 1460 as the min? 1460 is the min  MTU to carry IPv6. 576 I think is the old common min for IPv4.

Reply:
OK, but according to the IPv6 RFC, min MTU to carry IPv6 is 1280: https://datatracker.ietf.org/doc/html/rfc8200#section-5

Reply:
> OK, but according to the IPv6 RFC, min MTU to carry IPv6 is 1280: https://datatracker.ietf.org/doc/html/rfc8200#section-5

You're correct.